### DISCUSSION TITLE
Include option to save page as image

### DISCUSSION THREAD

**Discussion by lumnikemel** [gildas-lormeau/SingleFile#853] [Ideas]
Hi. I know this request is a little outside the wheelhouse of the extension, but I have not yet been able to find any screenshot-specific extension that also has the ability to automatically capture the page with having to hit a button. I archive all my browsing using this extension, but I would love to have screenshots of the pages, too. Especially for those pages that change content on me, like Twitter, if the page refreshes. I would want to have both the text and image evidence of deleted/suspended accounts.

Just a thought. Let me know if you are already aware of another solution to my problem.

Thanks.

**Comment by gildas-lormeau** [comment]
Unfortunately, it's not really in the scope of the extension. Besides, it's technically quite difficult to implement in Chrome because it doesn't offer the possibility to make a screenshot of the whole page. To do so, you'd have to implement some kind of auto-scroll, which is pretty awful from a user experience point of view.
